pkg/chartutil/values.go

```diff
@@ -390,7 +391,7 @@ func istable(v interface{}) bool {
 // PathValue takes a yaml path with . notation and returns the value if exists
```

Comment:
- [ ] Likewise, capitalize "YAML" here too, since you're covering this function in this patch.  
  It's not clear what a "YAML path" is, though. I think this comment is referring to something akin to field selector syntax applied to values read from a YAML file.